crates/hc_sandbox/Cargo.toml

```diff
@@ -68,8 +68,15 @@ workspace = true
 [features]
 default = []
 
-unstable-dpki = ["holochain_conductor_api/unstable-dpki"]
+unstable-dpki = [
+  "holochain_conductor_api/unstable-dpki",
+  "holochain_conductor_config/unstable-dpki",
+]
 
-chc = ["holochain_chc", "holochain_conductor_api/chc"]
+chc = [
+  "holochain_chc",
```

Comment:
```suggestion
  "dep:holochain_chc",
```

Reply:
Does not compile with this feature enabled

Reply:
Please don't resolve comments unless you're sure they're addressed. We don't have a clear way of working with that for the team, and it's something I'll raise for next Monday.

This was actually a request and then a problem report in two.

I'd like the suggestion to be applied because it means something different. I paused reviewing here because I ran into a build error but the initial comment was still wanted.